samza-core/src/main/java/org/apache/samza/clustermanager/ClusterBasedJobCoordinator.java

```diff
@@ -296,13 +310,9 @@ private StreamPartitionCountMonitor getPartitionCountMonitor(Config config, Syst
       throw new SamzaException("Input streams to a job can not be empty.");
     }
 
-    return new StreamPartitionCountMonitor(
-        inputStreamsToMonitor,
-        streamMetadata,
-        metrics,
-        new JobConfig(config).getMonitorPartitionChangeFrequency(),
-        streamsChanged -> {
-        // Fail the jobs with durable state store. Otherwise, application state.status remains UNDEFINED s.t. YARN job will be restarted
+    return new StreamPartitionCountMonitor(inputStreamsToMonitor, streamMetadata, metrics,
```

Comment:
no changes here in the diff right apart from indents?

Reply:
yes